apps/server/src/modules/tool/external-tool/controller/dto/request/config/lti11-tool-config-create.params.ts

```diff
@@ -32,4 +32,8 @@ export class Lti11ToolConfigCreateParams extends ExternalToolConfigCreateParams
 	@IsEnum(LtiPrivacyPermission)
 	@ApiProperty()
 	privacy_permission!: LtiPrivacyPermission;
+
+	@IsLocale()
```

Comment:
Does this accept the format which is required by lti ?

Reply:
yes